crates/cli/src/util.rs

```diff
@@ -344,13 +344,17 @@ pub fn host_or_url_to_host_and_protocol(host_or_url: &str) -> (&str, Option<&str
 /// Prompt the user for `y` or `n` from stdin.
 ///
 /// Return `false` unless the input is `y`.
-pub fn y_or_n(prompt: &str) -> anyhow::Result<bool> {
+pub fn y_or_n(force: bool, prompt: &str) -> anyhow::Result<bool> {
+    if force {
+        println!("Force flag is present, skipping confirmation");
+        return Ok(true);
+    }
     let mut input = String::new();
-    print!("{} (y/n)", prompt);
+    print!("{} [y/N]", prompt);
```

Comment:
The choice here of capital N was interesting here to me, then I asked chatgpt about it and this explanation makes sense

![image](https://github.com/user-attachments/assets/b2200f55-64ab-47b2-bcf1-5523a4e81157)

This styling seems exactly correct, well done Zeke 👍

Reply:
I copied it from one of the commands you made, so well done past-John :smile: